docs/configuration/basic_usage.md

```diff
@@ -117,6 +117,26 @@ my-service
 my-service-client
 ```
 
+Use the `foldersToExclude` configuration parameter to identify submodules
```

Comment:
I think this prominent feature should be also more visible in configuration. In general the long-term maintenance of this would be easier if there was:

* separate configuration object (features might pile up in the future)
* feature switch to allow using old flow without `null` or `isEmpty` checks

I suggest creating `MonorepoConfig` (transformed into `MonorepoProperties` to keep Gradle/domain isolation):

```
class MonorepoConfig {
    boolean enabled = false;
    List<String> projectDirs = [];
}

class VersionConfig {
    MonorepoConfig monorepo;
}
```